src/uikit.cs

```diff
@@ -1265,6 +1265,9 @@ interface NSLayoutManager : NSCoding {
 		[Export ("CGGlyphAtIndex:")]
 		ushort GetGlyph (nuint glyphIndex);
 		
+		[Watch (5, 0), TV (12, 0), iOS (12, 0)]
+		[Export ("limitsLayoutForSuspiciousContents")]
+		bool LimitsLayoutForSuspiciousContents { get; set; }
```

Comment:
that change is not mentioned in the commit message
it's also not reported by xtro - nor does it seems it was part of headers (yet?) for those platforms

Reply:
https://developer.apple.com/documentation/appkit/nslayoutmanager/3021179-limitslayoutforsuspiciouscontent?language=objc only mention macOS

Reply:
🤔 

This is from /Applications/Xcode10-beta4.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/AppKit.framework/Versions/C/Headers/NSLayoutManager.h

```// When YES, enables internal security analysis for malicious inputs and activates defensive behaviors. By enabling this functionality, it's possible certain text such as a very long paragraph might result in unexpected layout. NO by default.
@property BOOL limitsLayoutForSuspiciousContents API_AVAILABLE(macos(10.14), ios(12.0), watchos(5.0), tvos(12.0));
```

But you are correct that the NSLayoutManager.h in the iOS SDK does _not_ have it. 

That's what I get for trusting that header files common to different SDK actually are common. :(